pkg/graph/schema.resolvers.go

```diff
@@ -237,6 +237,27 @@ func (r *mutationResolver) DeleteDiscussionReply(ctx context.Context, id uuid.UU
 	return resolvers.DeleteDiscussionReply(logger, id, principal, r.store)
 }
 
+func (r *mutationResolver) LockTaskListSection(ctx context.Context, modelPlanID uuid.UUID, section string) (bool, error) {
+	principal := appcontext.Principal(ctx).ID()
+
+	resolvers.LockTaskListSection(r.pubsub, modelPlanID, section, principal)
+
+	return true, nil
+}
+
+func (r *mutationResolver) UnlockTaskListSection(ctx context.Context, modelPlanID uuid.UUID, section string) (bool, error) {
+	principal := appcontext.Principal(ctx).ID()
+
+	resolvers.UnlockTaskListSection(r.pubsub, modelPlanID, section, principal)
+
+	return true, nil
```

Comment:
I think this is potentially very confusing to always return true. 

I locked the task as USER MINT, tried to unlock as user COOL. I could see that it didn't unlock, but I got a response that suggested that it did unlock the task list section. I think maybe there should be a different return type for this struct? This also is swallowing the error
![image](https://user-images.githubusercontent.com/99294987/171964468-7026cb01-75fd-46b5-abdc-3e45a76f042f.png)

I think the paradigm for the other resolvers is to have the resolver return the return object and error, instead of building it here in the schema resolvers file.

Reply:
Great point. This is updated for Lock and Unlock along with descriptive errors for the cases they can fail. There is also improved feedback from the admin endpoint UnlockAllTaskListSections to return the list of section locks removed.